router/ui-routes.js

```diff
@@ -25,6 +27,17 @@ const errors = source("models/error");
 const Challenge = source("models/roboticon");
 const myMarked = require("marked");
 
+const execRoles = [
+    {
```

Comment:
Add the rest of the exec types, and a default email, each exec roll has an email address associated with it.

Reply:
what is the email address associated with each? I couldn't find them

Reply:
President -> `president@socis.ca`
VP Internal -> `vp-internal@socis.ca`
VP External -> `vp-external@socis.ca`
VP Finance -> `vp-finance@socis.ca`
VP Communications -> `vp-communications@socis.ca`
Secretary -> `secretary@socis.ca`
System Admin -> `admin@socis.ca`
First Year Representative  -> `first-year-rep@socis.ca`
Student Representative -> `student-rep@socis.ca`

Reply:
Added